src/System.Management.Automation/engine/SessionStateContainer.cs

```diff
@@ -3774,6 +3774,11 @@ internal Collection<PSObject> NewItem(string[] paths, string name, string type,
                 {
                     PSTraceSource.NewArgumentNullException("paths");
                 }
+                else if (path.EndsWith((":" + Path.DirectorySeparatorChar), StringComparison.Ordinal) || path.EndsWith((":" + Path.AltDirectorySeparatorChar), StringComparison.Ordinal))
```

Comment:
Might be worth hanging a line for the second path check.

Reply:
Agree, done.